translations/de.ts

```diff
@@ -1736,7 +1592,7 @@ Profile does not contain your history.</source>
         <location filename="../src/loginscreen.ui" line="354"/>
         <location filename="../src/loginscreen.ui" line="727"/>
         <source>Username:</source>
-        <translation>Spitzname:</translation>
+        <translation>Profilname:</translation>
```

Comment:
Hm ... in the above mentioned sentence (the one with "Chat-Verlauf"), the English version uses the term "profile". So it's ok for me as well.